src/tools/debugger/entities/BreakpointType.java

```diff
@@ -135,6 +135,15 @@ public void registerOrUpdate(final Breakpoints bps, final SectionBreakpoint bpIn
     }
   },
 
+  @SerializedName("activityOnExecBP")
+  ACTIVITY_ON_EXEC("activityOnExecBP", "On execution",
```

Comment:
No, as the breakpoint definition here hopefully states, it is for all operations tagged with `ActivityCreation`. And currently, that's the `ActivitySpawn` primitives. So, processes, tasks, threads.
It doesn't really make sense for actors though.